src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -556,11 +556,11 @@ private void InnerStartOperationSendPackets()
             {
                 if (spe != null)
                 {
-                    if (spe.m_FilePath != null)
+                    if (spe._filePath != null)
```

Comment:
Since these properties are internal and so public, shouldn't we have a different naming convention?
I understand replacing `this.` with `_` but having `object._member` looks odd.

I propose renaming _buffer to Buffer. This would be similar to how naming was done for `SocketAddress.InternalSize` and `SocketAddress.Buffer`.